test/controllers/confirm.company.controller.unit.ts

```diff
@@ -228,6 +228,17 @@ describe("Confirm company controller tests", () => {
     expect(response.text).toContain("You cannot use this service - File a confirmation statement");
   });
 
+  it("Should redirect to use webfiling stop screen when the eligibility status code is INVALID_COMPANY_APPOINTMENTS_MORE_THAN_FIVE_OFFICERS", async () => {
```

Comment:
I'd add a test for the 5 pscs status code as well

Reply:
Had left it to be covered as part of the page navigation ticket but might as well add the tests in here since i've claimed the responsibility of adding the status to the controller.